ted_sws/notice_packager/services/metadata_transformer.py

```diff
@@ -38,10 +38,6 @@ def normalize_value(cls, value: str) -> str:
     def denormalize_value(cls, value: str) -> str:
         return value.replace(NORM_SEP, DENORM_SEP)
```

Comment:
can we rename NORM_SEP and DENORM_SEP into something more meaningful and readable?

Reply:
Rename them how you wish. Names stand for DENORMALIZATION_SEPARATOR and NORMALIZATION_SEPARATOR